_posts/snapshot-management.md

```diff
@@ -0,0 +1,145 @@
+
+You need to set the schedule for your automatic snapshots using a [cron expression](https://opensearch.org/docs/latest/monitoring-plugins/alerting/cron/). Let's dissect the expression `0 8 * * *`, which we'll use in the example below.
+
+:--- | :--- | :--- | :--- | :---
+0 | 8 | * | * | *
+minute| hour | day of month | month | day of year
+
+This cron expression means you want to take snapshots at 8:00 AM every day.
+
+Here's a sample SM policy that puts together all of this information:
+
+```json
+POST _plugins/_sm/policies/daily-policy
+{
+  "description": "Daily snapshot policy",
+  "creation": {
+    "schedule": {
+      "cron": {
+        "expression": "0 8 * * *",
+        "timezone": "UTC"
+      }
+    },
+    "time_limit": "1h"
+  },
+  "deletion": {
+    "schedule": {
+      "cron": {
+        "expression": "0 1 * * *",
+        "timezone": "America/Los_Angeles"
+      }
+    },
+    "condition": {
+      "max_age": "7d",
+      "max_count": 21,
+      "min_count": 7
+    },
+    "time_limit": "1h"
+  },
+  "snapshot_config": {
+    "date_format": "yyyy-MM-dd-HH:mm",
+    "timezone": "America/Los_Angeles",
+    "indices": "*",
+    "repository": "s3-repo",
+    "ignore_unavailable": "true",
+    "include_global_state": "false",
+    "partial": "true",
+    "metadata": {
+      "any_key": "any_value"
+    }
+  },
+  "notification": {
+    "channel": {
+      "id": "NC3OpoEBzEoHMX183R3f"
+    },
+    "conditions": {
+      "creation": true,
+      "deletion": false,
+      "failure": false,
+      "time_limit_exceeded": false
+    }
+  }
+}
+```
+
+After you create an SM policy, you may start it using the [SM Start API](https://opensearch.org/docs/latest/opensearch/snapshots/sm-api/#start-a-policy) and stop it using the [SM Stop API](https://opensearch.org/docs/latest/opensearch/snapshots/sm-api/#stop-a-policy). To view the status of the snapshot operations, you can use the [SM Explain API](https://opensearch.org/docs/latest/opensearch/snapshots/sm-api/#explain).
+
+### What if something goes wrong?
+
+Snapshots are retried three times. Even if you don't set up notifications of failure in the SM policy, you can view the failure message using the [SM Explain API](https://opensearch.org/docs/latest/opensearch/snapshots/sm-api/#explain) or on the [policy details page in OpenSearch Dashboards](https://opensearch.org/docs/latest/opensearch/snapshots/sm-dashboards#view-edit-or-delete-an-sm-policy).
+
+### How does SM work under the hood?
+
+SM uses a state machine that has two workflows: creation and deletion. These workflows are executed sequentially, so SM does not start a new operation until the previous operation finishes. 
+
+The following image shows the creation workflow of the SM state machine.
+
+<img src="/assets/media/blog-images/2022-09-16/sm-state-machine.PNG" alt="SM State Machine" width="150" style="float: left; margin-right: 15px;"/>
```

Comment:
please fix the path on this one - should be:
assets/media/blog-images/**2022-09-16-snapshot-management**/sm-state-machine.PNG

Reply:
@kolchfa-aws - you will also need to create an author bio and image as part of this post. thanks